api/src/main/java/jakarta/data/repository/Repository.java

```diff
@@ -169,6 +169,11 @@
  * match the parameter value, which can be a pattern.</td>
  * <td><code>findByNameEndsWith(surname)</code></td></tr>
  *
+ * <tr style="vertical-align: top"><td><code>False</code></td>
+ * <td>boolean</td>
+ * <td>Requires that the entity's attribute value has a boolean value of false.</td>
+ * <td><code>findByIsCanceledFalseAndDurationLessThan(max)</code></td></tr>
```

Comment:
The sample style is confusing. Let's have only `findByCanceledFalse`. `AndDurationLessThan` creates quite some noise. Also, `IsCanceled` would apply if the property is named `isCanceled` (`private boolean isCanceled`).

Same comment applies for the `True` and `null` samples.

Reply:
I missed the rename from `IsCanceled` to `Canceled`, so there's also this commit https://github.com/jakartaee/data/pull/96/commits/aad44dfa048f1b889ddf709134689b1485a9ab93 to cover that.